jams/core.py

```diff
@@ -496,49 +498,21 @@ class Sandbox(JObject):
     pass
 
 
-class JamsFrame(pd.DataFrame):
-    '''A data-frame class for JAMS.
+Observation = namedtuple('Observation',
+                         ['time', 'duration', 'value', 'confidence'])
+'''Core observation type: (time, duration, value, confidence).'''
 
-    This automates certain niceties, such as timestamp
-    conversion and serialization.
-    '''
-
-    __dense = False
-
-    def __init__(self, data=None, index=None, columns=None, dtype=None):
-        '''Construct a new JamsFrame object.
-
-        Parameters
-        ----------
-        data
-            Optional data for the new JamsFrame, in any format supported
-            by `pandas.DataFrame.__init__`.
 
-            Fields must be `['time', 'duration', 'value', 'confidence']`.
+class AnnotationData(object):
```

Comment:
Thinking more about this:

Another option is to drop `AnnotationData` entirely, keep `ann.data` as a sorted container object, and bubble up the rest of the functionality for adding observations, iterations, etc to `Annotation`.  This would be a more radical change in the API than I'd originally intended, but it seems like maybe a good idea provided we're willing to break stuff anyway.